index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!-- Page Title -->
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <!-- Description -->
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+
```

Comment:
- [ ] For semantic reasons please ensure to add the <header> section and wrap all the corresponding content inside it.